test/unit/libp2p_test.exs

```diff
@@ -2,6 +2,7 @@ defmodule Libp2pTest do
   use ExUnit.Case
   doctest Libp2p
 
+  # Mainnet
   @bootnodes [
```

Comment:
If this is an external ip then this should probably go into "test/integration", right?

Reply:
Those are the Mainnet bootnodes' addresses.